docs/requirements.txt

```diff
@@ -2,6 +2,7 @@ sphinx
 sphinx-autobuild
 sphinx-autoapi
 groundwork-sphinx-theme
+sphinx_copybutton
```

Comment:
Consider also adding these here, aligning with Sphinx configuration dependencies.
```suggestion
sphinx_copybutton
m2r2
furo
```

Reply:
Great!

Reply:
oh! I didn't realize this was a suggestion. Added myself in 787905ac9e9a451c66ff7e2720bb95e7700422d3 oops